src/messages/abci.rs

```diff
@@ -964,17 +970,17 @@ impl ::protobuf::Message for Request {
 
 impl ::protobuf::Clear for Request {
     fn clear(&mut self) {
-        self.clear_echo();
-        self.clear_flush();
-        self.clear_info();
-        self.clear_set_option();
-        self.clear_init_chain();
-        self.clear_query();
-        self.clear_begin_block();
-        self.clear_check_tx();
-        self.clear_deliver_tx();
-        self.clear_end_block();
-        self.clear_commit();
+        self.value = ::std::option::Option::None;
```

Comment:
I guess updating to the latest protobuf changed these?

Reply:
Yes, these were all updated by the protobuf library